web/src/components/molecules/Schema/FieldModal/FieldUpdateModal/index.tsx

```diff
@@ -235,12 +244,25 @@ const FieldUpdateModal: React.FC<Props> = ({
             )}
             <Form.Item
               name="multiValue"
+              valuePropName="checked"
               extra={t("Stores a list of values instead of a single value")}>
               <Checkbox>{t("Support multiple values")}</Checkbox>
             </Form.Item>
           </TabPane>
           <TabPane tab={t("Validation")} key="validation" forceRender>
             <FieldValidationInputs selectedType={selectedType} />
+            <Form.Item
+              name="required"
+              valuePropName="checked"
+              extra={t("Prevents saving an entry if this field is empty")}>
+              <Checkbox>{t("Make field required")}</Checkbox>
+            </Form.Item>
+            <Form.Item
+              name="unique"
+              valuePropName="checked"
+              extra={t("Ensures that a multiple entries can't have the same value for this field")}>
```

Comment:
Ensures that multiple entries can't have the same value for this field

Reply:
@KaWaite, updated.